app/src/main/java/com/github/campus_capture/bootcamp/firebase/PlaceholderBackend.java

```diff
@@ -1,5 +1,20 @@
 package com.github.campus_capture.bootcamp.firebase;
 
+import static com.github.campus_capture.bootcamp.authentication.Section.AR;
```

Comment:
A `import static com.github.campus_capture.bootcamp.authentication.Section.*;` would maybe be cleaner but it is also totally fine.